src/containers/setContacts/actions.ts

```diff
@@ -0,0 +1,8 @@
+import { genericAsyncActions } from '../../utils/asyncRequest';
+
+export const setContacts = genericAsyncActions<void, void>();
+
+export type SetContactsActions =
+    | ReturnType<typeof setContacts.loading>
+    | ReturnType<typeof setContacts.loaded>
```

Comment:
@anirudh723 I dont really understand what is means for a POST request that sets information to be "Loaded", why do we need redux for this?

Reply:
Yeah I don't think much of my recent commit was correct... feel free to revert it back